### PR TITLE
Update gan practice

### PR DESCRIPTION
**Pull request by sovrasov**
[Updated model](https://drive.google.com/file/d/1a1lv9rZXh-ydECfvfF1K0DFSafhPgp25/view?usp=sharing)

### CODE REVIEW

**Comment by dkurt** [comment]
@sovrasov, the link is only for `.xml`

**Comment by sovrasov** [comment]
[Correct link](https://drive.google.com/drive/folders/18lDnBUtd_mgQHn_ckaUwERwU2IS9I4gy?usp=sharing)

**Comment by dkurt** [comment]
Please update the links in `.travis.xml` and `README.md`:

https://mega.nz/#!pUN1kCCQ!e9foW8shMj6fpA7go8_RjtGo5YGYKtiI14SVCSop_l0

https://mega.nz/#!RcUHRaQI!-4XReh2kFr_sgvl8qmKkVhkhTcxEWDVXMP1xFv8fsp8

File: .travis.yml [hunk 1]
```diff
@@ -44,8 +44,8 @@ before_install:
   - megadl 'https://mega.nz/#!RAFU1I4T!xjuol1lZ9t_hA67AxgAxp8w1JIb5FXm53617UXR0NO8' --path data --no-progress  # frozen_unet_histology.pb
   # TODO: Implement frozen_unet_histology.pb model conversion with OpenVINO
   # (note that TensorFlow accepts input tensor of shape 1,952,1144,1,1,3)
-  - megadl 'https://mega.nz/#!AR1HwIIA!U3p46tM9SFMhKzbSwJIEiugdJDQ5mUSOC3twZL6UYTM' --path data --no-progress  # celeba_generator.xml
-  - megadl 'https://mega.nz/#!hAslCCaL!2vWXjiPq3EbuLZbOvEDOI0y9VorGn5enUm7UMNWGrWo' --path data --no-progress  # celeba_generator.bin
+  - megadl 'https://mega.nz/file/RcUHRaQI#-4XReh2kFr_sgvl8qmKkVhkhTcxEWDVXMP1xFv8fsp8' --path data --no-progress  # celeba_generator.xml
+  - megadl 'https://mega.nz/file/pUN1kCCQ#e9foW8shMj6fpA7go8_RjtGo5YGYKtiI14SVCSop_l0' --path data --no-progress  # celeba_generator.bin
```
Comment (dkurt) [inline .travis.yml]: Please use direct links (from https://github.com/dkurt/openvino_practice/pull/216#issuecomment-659345547)

**Pull request merged**